dbms/src/Common/FieldVisitors.h

```diff
@@ -313,7 +328,7 @@ class FieldVisitorAccurateLess : public StaticVisitor<bool>
     bool operator() (const Int64 & l, const Tuple & r)      const { return cantCompare(l, r); }
     bool operator() (const Int64 & l, const AggregateFunctionStateData & r) const { return cantCompare(l, r); }
 
-    bool operator() (const Float64 & l, const Null & r)     const { return cantCompare(l, r); }
+    bool operator() (const Float64 &, const Null &)         const { return NULLS_LAST; }
     bool operator() (const Float64 & l, const UInt64 & r)   const { return accurate::lessOp(l, r); }
     bool operator() (const Float64 & l, const UInt128 & r)  const { return cantCompare(l, r); }
     bool operator() (const Float64 & l, const Int64 & r)    const { return accurate::lessOp(l, r); }
```

Comment:
Also look at line 342, maybe it was incorrect.
```
template <typename T>
bool operator() (const Null &, const T &) const
```

Reply:
Yes, I forgot about this method, thanks.